_posts/2020-07-23-what-was-one-crisis-is-now-policy.md

```diff
@@ -0,0 +1,29 @@
+layout : blog
+
+title : “What Was Once Crisis Is Now Policy”
+
+description : The pandemic made you adapt and adlib. What stuck?
+
+image : 2020-07-23-what-was-once-crisis-is-now-policy.jpg
+
+author : John Scott
+
+comments : true
+
+tags : [John Scott, Software,COVID-19]
+---
+
+<p>For most of us, we're at week 18 of this global health battle. It feels like it's been forever.</p>
+<p>We likely initially panicked, then furloughed/laid off people. We scrubbed everything in sight. Some of got a mask. Some of us didn't. We cut back, slimmed down, or maybe even staffed up. My wife works for an EdTech firm, deploying distance learning software solutions for K-12. She can barely keep up.</p>
```

Comment:
"Some of got a mask." should be "Some of us got a mask."

Reply:
I noticed a couple of little tweaks needed:
- you'll need to add '---' to the top of this file to get the page to render your image and fill in all the variables you entered.
- the word adlib should be hyphenated if it's being used as a verb or adjective: 'ad-lib'
- the name of the file says '...one-crisis...' but I think it should say '...once-crisis...'